backend/ormconfig.js

```diff
@@ -18,8 +18,8 @@ module.exports = [
   {
     type: 'postgres',
     url: process.env.DATABASE_URL,
-    ssl: sslConfig.ssl,
-    extra: sslConfig.extra,
+    ssl: prodSSLConfig.ssl,
```

Comment:
why are we changing this?

Reply:
This was originally there for me personally when connecting with the dev DB. Before I finish this PR I plan on going ahead and reverting this file back. I use npm instead of docker